docs/source/reference/glossary.rst

```diff
@@ -1,6 +1,72 @@
 Glossary
 ========
 
+
+API
+---
+
+application programming interface
```

Comment:
Can we add to this: "MatFlow's API allows us to design and execute workflows from within a Python interpreter or Jupyter notebook."